apps/build.sh

```diff
@@ -175,10 +175,14 @@ for x in $IMAGES ; do
 
 	echo "Build step $((completed+2)) of $total is complete"
 
-	status "Doing a syft SBOM scan"
-	sbom_dst=/archive/sboms/${ct_base}/${ARCH}.spdx.json
-	mkdir -p $(dirname $sbom_dst)
-	syft ${ct_base}:$TAG-$ARCH -o spdx-json > $sbom_dst
+	if [ -z "$DISABLE_SBOM" ] ; then
```

Comment:
@doanac To implement this a user adds this to the `params` block of `containers`?
```
DISABLE_SBOM: "1"
```

Reply:
yep. i'm testing that now

Reply:
here it is disabled - https://app.foundries.io/factories/andy-corp/targets/77/ 
here it is enabled(default) - https://app.foundries.io/factories/andy-corp/targets/78/